giovanniortega/react-apprenticeship-capstone1
# Feat/capstone project 1/minichallenge 1 2

## Description
```
< Mini Challenge 1 - 2>

- New app created, Home page, Header, Search, Favorites, Login modal added, 
the app is fetching the test data and printing content in the home page .
All the Linter errors were added
Some react testing cases were added.

```
## Type of Change

New Features / development

```
Delete options that are not relevant.
```


## How should this be tested?
```
Only Basic tests cases were added to verify if the content is in the document, 
just run 'npm run test' and 'npm test -- --coverage --watchAll=false '

```
## Any background context you want to provide?
```
This is only a new development, there are additional features developed that weren't  required in the first mini challenges instructions like the login modal, the favorites section and the search component structures
```
## What is the deliverable?
```
Mini Challenge 1 - 2
```

## Checklist
